docs/concepts/macros/sqlmesh_macros.md

```diff
@@ -260,10 +260,22 @@ It includes three elements:
 2. A value to return if the condition is `TRUE`
 3. A value to return if the condition is `FALSE` [optional]
 
-These elements are specified as `@IF([logical condition], [value if TRUE], [value if FALSE])`.
+These elements are specified as:
+
+```sql linenums="1"
+@IF([logical condition], [value if TRUE], [value if FALSE])
+```
 
 The value to return if the condition is `FALSE` is optional - if it is not provided and the condition is `FALSE`, then the macro has no effect on the resulting query.
 
+It's also possible to use this macro in order to conditionally execute pre/post-statements:
```

Comment:
Is it possible to have both unconditional and conditional pre/post statements? Like if I have one pre-statement that should always run and a second pre-statement that should only run sometimes?

Reply:
Yeah that's correct. All pre/post-statements are always rendered, but the ones wrapped in `@IF` are simply rendered to `None` IIRC and so they won't included in the execution? I'll need to double check that.